.rubocop_todo.yml

```diff
@@ -202,7 +207,17 @@ Naming/UncommunicativeMethodParamName:
 Naming/VariableNumber:
   Enabled: false
 
-# Offense count: 198
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/Count:
+  Enabled: false
+
+# Offense count: 3
+# Cop supports --auto-correct.
+Performance/Detect:
+  Enabled: false
+
```

Comment:
These look like cops we fixed but new version found some additional cases that weren't detected before?

Reply:
Probably, yes... I'll autocorrect...